src/background.ts

```diff
@@ -145,6 +145,11 @@ if (!fs.existsSync(vvppEngineDir)) {
   fs.mkdirSync(vvppEngineDir);
 }
 
+const exportEngineInfoFilename = path.join(
```

Comment:
RuntimeInfoを良い感じに書けるよう、クラスにしてあげた方が良い気がしますね･･･。
いけそうだったら実装お願いしたいかもです！（ConfigManagerみたいな感じでRuntimeInfoManagerを作る感じ）
難しそうであればとりあえずFIXMEコメントを残しておければ！
```suggestion
// FIXME: RuntimeInfoクラスにする
const exportEngineInfoFilename = path.join(
```

Reply:
ありがとうございます。クラス化してみようと想います。
ファイル位置を shared配下にしようと想いましたが、他に適切な場所があれば
お勧め頂けるとたすかります（階層をどうやって判断するのか、判断基準が難しい）

Reply:
判断に関しては僕もとても迷います･･･。もし提案あれば別でコメントします！